routes/index.js

```diff
@@ -1,10 +1,15 @@
 var express = require("express");
-var router = express.Router();
+const path = require('path');
+
+const app = express();
+
+//app.use('/api');
+app.use(express.static(path.join(__dirname, 'api')));
```

Comment:
This is for pulling static content from Express. I think we need to point to '/pulic' here for files like page not found. This isn't particularly relevant since you're using Express as an API only, but still probably a good idea to switch to `public`